src/libraries/System.Runtime.InteropServices/gen/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -392,7 +392,7 @@ private static IncrementalStubGenerationContext CalculateStubInformation(MethodD
             AttributeData? lcidConversionAttr = null;
             AttributeData? suppressGCTransitionAttribute = null;
             AttributeData? unmanagedCallConvAttribute = null;
-            foreach (var attr in symbol.GetAttributes())
+            foreach (AttributeData? attr in symbol.GetAttributes())
```

Comment:
```suggestion
            foreach (AttributeData attr in symbol.GetAttributes())
```

Reply:
I'll stop commenting on all these.  Looks like many of the substitutions don't actually need to be nullable.  I believe the IDE automatically makes them nullable in case you might want to assign something null later (a design decision I don't agree with).

Reply:
Ah, boo. Thanks.